release-controller/reconciler.py

```diff
@@ -57,17 +58,27 @@ def version_proposal(self, version: str) -> int | None:
 
     def proposal_submitted(self, version: str) -> bool:
         """Check if a proposal has been submitted for the given version."""
+        version_path = self._version_path(version)
         if self._version_path(version).exists():
             proposal_id = self.version_proposal(version)
             if proposal_id:
                 logging.info(
                     "version %s: proposal %s already submitted", version, proposal_id
                 )
             else:
-                logging.warning(
-                    "version %s: earlier proposal submission attempted but failed, will not retry",
-                    version,
+                last_modified = datetime.datetime.fromtimestamp(os.path.getmtime(version_path))
+                remaining_time_until_retry = datetime.timedelta(minutes=3) - (
```

Comment:
```suggestion
                remaining_time_until_retry = datetime.timedelta(minutes=10) - (
```

to be on a safer side